packages/dht/src/connection/websocket/WebsocketServerConnection.ts

```diff
@@ -26,14 +26,19 @@ export class WebsocketServerConnection extends EventEmitter<ConnectionEvents> im
 
         this.resourceURL = resourceURL
         this.connectionId = createRandomConnectionId()
-        this.remoteIpAddress = remoteAddress
+        this.remoteAddress = remoteAddress
 
         socket.on('message', this.onMessage)
         socket.on('close', this.onClose)
         socket.on('error', this.onError)
 
         this.socket = socket
     }
+    
+    // use a getter to make it possible to mock the value in tests
+    public get remoteIpAddress(): string {
```

Comment:
```suggestion
    public getRemoteIpAddress(): string {
```

Reply:
The use of an actual js getter is intentional, the user of WebSocketServerConnection does not need to know this field has been changed into a getter.

Reply:
change to java-style getter function